Editor/Deployment/DeploymentSettings.cs

```diff
@@ -253,6 +248,10 @@ public void Save()
             _coreApi.PutSettingOrClear(SettingsKeys.DeploymentBuildFolderPath, BuildFolderPath);
             _coreApi.PutSettingOrClear(SettingsKeys.DeploymentBuildFilePath, BuildFilePath);
             _coreApi.PutSettingOrClear(SettingsKeys.DeploymentGameName, GameName);
+            _coreApi.PutSettingOrClear(SettingsKeys.LaunchParameters, LaunchParameters);
+            _coreApi.PutSettingOrClear(SettingsKeys.BuildOperatingSystem, BuildOperatingSystem);
+            _coreApi.PutSettingOrClear(SettingsKeys.FleetName, FleetName);
+            _coreApi.PutSettingOrClear(SettingsKeys.BuildName, BuildName);
```

Comment:
Same as the previous comment about reusing the `Settings` wrapper for this.

Reply:
see previous comment